tests/drawing/test_draw.py

```diff
@@ -183,3 +183,41 @@ def test_draw_hypergraph_hull(edgelist8):
     assert ax.collections[0].get_zorder() == 4  # nodes
 
     plt.close()
+
+
+def test_draw_multilayer(edgelist8):
```

Comment:
```suggestion
def test_correct_number_of_collections_draw_multilayer(edgelist8):
```

@leotrs thanks for the comment, you mean something like this?

Reply:
Perfect!